gamemodes/terrortown/gamemode/server/sv_networking.lua

```diff
@@ -481,3 +481,24 @@ local function ttt_roles_index(ply)
 	ply:ChatPrint("----------------")
 end
 concommand.Add("ttt_roles_index", ttt_roles_index)
+
+---
```

Comment:
A small comment should be added, if we are already adding the general documentation.